docs/Horizon/Horizon-overview.md

```diff
@@ -8,69 +8,277 @@
   - further granular RwaAToken admin permissions will be configured in the RwaATokenManager contract itself.
-  - Token Issuers or relevant admin will be granted admin permissions on the RwaAToken corresponding to their specific RWA asset.
+  - Token Issuers or relevant admin can be granted admin permissions on the RwaAToken corresponding to their specific RWA asset.
 - No bridges/portals will be configured, hence no unbacked RwaATokens can be minted. 
 
-#### Edge Cases of Note
+#### Reserve
 
-- User has a borrow position but loses private keys to wallet. This position will need to be migrated to a new wallet. Issuers can resolve using:
-  - authorized flashborrow to borrow enough stablecoin to repay a user's debt.
-  - repay `onBehalfOf` to repay debt on behalf of user.
-  - `ATOKEN_ADMIN` to move RwaAToken collateral to new wallet.
-  - open a new borrow position from new wallet.
-- User creates a position in Horizon but then becomes sanctioned. Their actions will need to be blocked until further resolution. Issuers can resolve using:
-  - `ATOKEN_ADMIN` to move maximum allowable RwaAToken collateral to temporary wallet, preventing further borrowing.
-  - prevent the liquidation of the sanctioned user's position through off-chain coordination.
+- priceFeed: different per asset, but will be required to be Chainlink-compatible
+- rateStrategyParams: N/A (can be left empty)
+- borrowingEnabled: false (to prevent its borrowing)
+- borrowableInIsolation: false
+- withSiloedBorrowing: false
+- flashloanable: false
+- LTV: different per asset, <100%
+- liqThreshold: different per asset, <100%
+- liqBonus: different per asset, >100%
+- reserveFactor: 0
+- supplyCap: different per asset
+- borrowCap: 0
+- debtCeiling: non-zero if the RWA asset is in isolation
+- liqProtocolFee: 0 (must be 0, otherwise liquidations will revert in RwaAToken due to `transferOnLiquidation`)
 
 ### Stablecoins (Borrowable Asset)
 
-Stablecoins can be supplied permissionlessly to earn yield. However, they will only be able to be borrowed, but disabled as collateral assets (via asset configuration, by setting LTV to 0). Borrowing will be implicitly permissioned because only users that have supplied RWA assets can borrow stablecoins. Other existing functionality remains the same as in v3.3. Stablecoin assets will be listed as usual, also working in a standard way.
+Stablecoins can be supplied permissionlessly to earn yield. However, they will only be able to be borrowed, but disabled as collateral assets (via asset configuration, by setting Liquidation Threshold to 0). Borrowing will be implicitly permissioned because only users that have supplied RWA assets can borrow stablecoins (except in a potential case [here](#non-allowlisted-account-can-receive-rwaatokens)). Other existing functionality remains the same as in v3.3. Stablecoin assets will be listed as usual, also working in a standard way.
```

Comment:
```suggestion
Stablecoins can be supplied permissionlessly to earn yield. However, they will only be able to be borrowed, but disabled as collateral assets (via asset configuration, by setting Liquidation Threshold to 0). Borrowing will be implicitly permissioned because only users that have supplied RWA assets can borrow stablecoins (except in a potential case [here](#non-allowlisted-account-can-receive-rwaatokens)). 
All other existing functionality remains unchanged from v3.3. Stablecoin assets will be listed and operate as usual, following the standard process.
```